src/Orleans/Runtime/OutsideRuntimeClient.cs

```diff
@@ -65,7 +65,8 @@ internal class OutsideRuntimeClient : IRuntimeClient, IDisposable
         private GrainInterfaceMap grainInterfaceMap;
         private readonly ThreadTrackingStatistic incomingMessagesThreadTimeTracking;
 
-        private static readonly TimeSpan initTimeout = TimeSpan.FromSeconds(10);
+        // initTimeout used to be AzureTableDefaultPolicies.TableCreationTimeout, whcih was 3 min
+        private static readonly TimeSpan initTimeout = TimeSpan.FromMinutes(3);
```

Comment:
3 minutes seems like an awfully long time to wait during initialization. Shouldn't we shorten it dramatically?